### PR TITLE
Give roundstart cultists inherent .x

### PR DESCRIPTION
**Pull request by Allspice**
0% tested for now since my local server's bugging out. A concession to the crew can be added if this is "pushing the envelope" too much.

:cl:
* tweak: Give roundstart cultists inherent .x

### CODE REVIEW

**Comment by SonixApache** [comment]
\>no more need to meet in secret without raising suspicion
\>just scream over the purple radio about your plans
yes very culty

i have some other good ideas for cult
 - re-usable conversion item that fits in your pocket
 - cool sunglasses
 - thing that lets you remove walls, like some sorta plastic explosives
 - gun (but it's blood)

**Comment by Irradiation** [comment]
So what's the point of the Cult Comms tattoo if it becomes an inherent ability?

**Comment by Allspice** [comment]
Tattoo is for the non-roundstart cultists.

**Comment by Spyroshark** [comment]
Having to meet up in secluded places and whisper to each other is one of the only parts of cult that actually feels like it's a cult, why would we want to get rid of that?

**Comment by Allspice** [comment]
A number of reasons, main one being that I've heard/experienced enough cult rounds where some new dude did something stupid in an obvious place and every other cultist eats shit from it. At least with the roundstart culters getting comms they can ask who all's new and tell them what dumb shit not to do.

**Comment by juanototo2** [comment]
It's time to end illegal W once and for all.

**Comment by SonixApache** [comment]
I understand your point, Yred, but I think this isn't the best way of approaching it.

**Comment by ShiftyRail** [comment]
Here is a five part plan on how to make cult great : 
1. Head cultists with communication papers and inherent .x (if needed).
2. Remove bases in deep space.
3. Only one Pale Body for the whole cult, but to compensate, he gets all the other T3 spells too.
4. Being able to respond to a .x meeting without drawing a rune yourself. Some short message which only gets to the guy doing the ritual.
5. Removing converting through implants

**Comment by Allspice** [comment]
@SonixApache The ways the other team antags "manage" the situation is by giving the members their own chat (ops, xeno, pox, defsquad, revsquad) or spawning them in the same location (wiz+apprentice or vamp+thralls I guess). Can't say much about revs specifically since I haven't played in a rev round since the alerting change.

File: code/modules/mob/living/say.dm [hunk 1]
```diff
@@ -486,9 +486,10 @@ var/list/department_radio_keys = list(
 	if(setting == 0) //overridden for constructs
 		return
 
-	if (iscultist(src))
+	var/datum/role/cultist/cultist = M.mind.GetRole(CULTIST)
```
Comment (ShiftyRail) [inline code/modules/mob/living/say.dm:489]: Remove `M`.

**Comment by Blithering** [comment]
Why not just bring back the starting supply paper in some form to give starting cultists some early game steam?

**Comment by tuypo1** [comment]
Maybe an argument could be made for making it inherent in act 1 so that the other tattoos actually see some use but roundstart is too much

**Comment by dunsel-hordon** [comment]
Still sounds like a good idea to have just the starting cultist have .x but there needs to be a downside to it for the community to accept this change.

**Comment by DeityLink** [comment]
I really don't think not having :x at roundstart is such a handicap for early round cultists, having to hide to write runes to communicate is part of the experience. This was also done to encourage cultists actually meeting together.

**Pull request closed without merging**